src/content/developers/tutorials/how-to-write-and-deploy-an-nft.md

```diff
@@ -0,0 +1,334 @@
+
+    888    888                      888 888               888
+    888    888                      888 888               888
+    888    888                      888 888               888
+    8888888888  8888b.  888d888 .d88888 88888b.   8888b.  888888
+    888    888     "88b 888P"  d88" 888 888 "88b     "88b 888
+    888    888 .d888888 888    888  888 888  888 .d888888 888
+    888    888 888  888 888    Y88b 888 888  888 888  888 Y88b.
+    888    888 "Y888888 888     "Y88888 888  888 "Y888888  "Y888
+    👷 Welcome to Hardhat v2.0.11 👷‍
+    ? What do you want to do? …
+    Create a sample project
+    ❯ Create an empty hardhat.config.js
+    Quit
+
+This will generate a hardhat.config.js file for us which is where we’ll specify all of the set up for our project (on step 13).
+
+## Step 9: Add project folders
+
+To keep our project organized, we’ll create two new folders. Navigate to the root directory of your project in your command line and type:
+
+    mkdir contracts
+    mkdir scripts
+
+* contracts/ is where we’ll keep our NFT smart contract code
+
+* scripts/ is where we’ll keep scripts to deploy and interact with our smart contract
+
+## Step 10: Write our contract
+
+Now time that our environment is set up, onto more exciting stuff: *writing our smart contract code!*
+
+Open up the my-nft project in your favorite editor (we like [VSCode](https://code.visualstudio.com/)). Smart contracts are written in a language called Solidity which is what we will use to write our MyNFT.sol smart contract.‌
+
+1. Navigate to the “contracts” folder and create a new file called MyNFT.sol
+
+1. Below is our NFT smart contract code, which based off of the [OpenZepplin](https://docs.openzeppelin.com/contracts/3.x/erc721) library’s ERC721 implementation. Copy and paste the contents below into your MyNFT.sol file.
+
+    //Contract based on [https://docs.openzeppelin.com/contracts/3.x/erc721](https://docs.openzeppelin.com/contracts/3.x/erc721)
+    // SPDX-License-Identifier: MIT
+    pragma solidity ^0.7.3;
+
+    import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+    import "@openzeppelin/contracts/utils/Counters.sol";
+    import "@openzeppelin/contracts/access/Ownable.sol";
+    
+
+    contract MyNFT is ERC721, Ownable {
+        using Counters for Counters.Counter;
+        Counters.Counter private _tokenIds;
+
+        constructor() public ERC721("MyNFT", "NFT") {}
+
+        function mintNFT(address recipient, string memory tokenURI)
+            public onlyOwner
+            returns (uint256)
+        {
+            _tokenIds.increment();
+
+            uint256 newItemId = _tokenIds.current();
+            _mint(recipient, newItemId);
+            _setTokenURI(newItemId, tokenURI);
+
+            return newItemId;
+        }
+    }
+
+3. Because we are inheriting classes from the OpenZepplin contracts library, in your command line run npm install @openzeppelin/contracts to install the library into our folder.
```

Comment:
Again, would add in some inline code snippet styling with backticks to help these commands stand out.